Index: module_4_2.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/module_4_2.py b/module_4_2.py
--- a/module_4_2.py	(revision d142844b39303e2e4d2de844edf9abf664942e46)
+++ b/module_4_2.py	(date 1724589862256)
@@ -1,0 +1,7 @@
+def test_function():
+    def inner_function():
+        print ("Я в области видимости функции test_function")
+    inner_function()
+
+test_function()
+inner_function()
\ No newline at end of file
Index: module_5_2.py
===================================================================
diff --git a/module_5_2.py b/module_5_2.py
deleted file mode 100644
--- a/module_5_2.py	(revision d142844b39303e2e4d2de844edf9abf664942e46)
+++ /dev/null	(revision d142844b39303e2e4d2de844edf9abf664942e46)
@@ -1,31 +0,0 @@
-class House:
-    def __init__(self, name:str, number_of_floors:int):
-        self.name = name
-        self.number_of_floors = number_of_floors
-
-    def __len__(self):
-        return self.number_of_floors
-    def __str__(self):
-        return self.name
-    def go_to (self, new_floors:int):
-        if new_floors > self.number_of_floors:
-            print ("Такого этажа не существует")
-        else:
-            for i in range(1,new_floors+1):
-                print (i)
-
-# h1 = House('ЖК Горский',18)
-# h2 = House('Домик в деревне',2)
-# h1.go_to(5)
-# h2.go_to(10)
-
-h1 = House('ЖК Эльбрус', 10)
-h2 = House('ЖК Акация', 20)
-
-# __str__
-print(f'Название: {h1.__str__()}, кол-во этажей: {h1.__len__()}.')
-print(f'Название: {h2.__str__()}, кол-во этажей: {h2.__len__()}.')
-
-# __len__
-print(len(h1))
-print(len(h2))
\ No newline at end of file
Index: module_5_1.py
===================================================================
diff --git a/module_5_1.py b/module_5_1.py
deleted file mode 100644
--- a/module_5_1.py	(revision d142844b39303e2e4d2de844edf9abf664942e46)
+++ /dev/null	(revision d142844b39303e2e4d2de844edf9abf664942e46)
@@ -1,15 +0,0 @@
-class House:
-    def __init__(self, name:str, number_of_floors:int):
-        self.name = name
-        self.number_of_floors = number_of_floors
-    def go_to (self, new_floors:int):
-        if new_floors > self.number_of_floors:
-            print ("Такого этажа не существует")
-        else:
-            for i in range(1,new_floors+1):
-                print (i)
-
-h1 = House('ЖК Горский',18)
-h2 = House('Домик в деревне',2)
-h1.go_to(5)
-h2.go_to(10)
\ No newline at end of file
Index: module_5_1_my.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/module_5_1_my.py b/module_5_1_my.py
new file mode 100644
--- /dev/null	(date 1724589862420)
+++ b/module_5_1_my.py	(date 1724589862420)
@@ -0,0 +1,32 @@
+<<<<<<< HEAD
+class House:
+    def __init__(self, name:str, number_of_floors:int):
+        self.name = name
+        self.number_of_floors = number_of_floors
+    def go_to (self, new_floors:int):
+        if new_floors > self.number_of_floors:
+            print ("Такого этажа не существует")
+        else:
+            for i in range(1,new_floors+1):
+                print (i)
+
+h1 = House('ЖК Горский',18)
+h2 = House('Домик в деревне',2)
+h1.go_to(5)
+=======
+class House:
+    def __init__(self, name:str, number_of_floors:int):
+        self.name = name
+        self.number_of_floors = number_of_floors
+    def go_to (self, new_floors:int):
+        if new_floors > self.number_of_floors:
+            print ("Такого этажа не существует")
+        else:
+            for i in range(1,new_floors+1):
+                print (i)
+
+h1 = House('ЖК Горский',18)
+h2 = House('Домик в деревне',2)
+h1.go_to(5)
+>>>>>>> origin/master
+h2.go_to(10)
\ No newline at end of file
Index: module_5_2_my.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/module_5_2_my.py b/module_5_2_my.py
new file mode 100644
--- /dev/null	(date 1724589862439)
+++ b/module_5_2_my.py	(date 1724589862439)
@@ -0,0 +1,64 @@
+<<<<<<< HEAD
+class House:
+    def __init__(self, name:str, number_of_floors:int):
+        self.name = name
+        self.number_of_floors = number_of_floors
+
+    def __len__(self):
+        return self.number_of_floors
+    def __str__(self):
+        return self.name
+    def go_to (self, new_floors:int):
+        if new_floors > self.number_of_floors:
+            print ("Такого этажа не существует")
+        else:
+            for i in range(1,new_floors+1):
+                print (i)
+
+# h1 = House('ЖК Горский',18)
+# h2 = House('Домик в деревне',2)
+# h1.go_to(5)
+# h2.go_to(10)
+
+h1 = House('ЖК Эльбрус', 10)
+h2 = House('ЖК Акация', 20)
+
+# __str__
+print(f'Название: {h1.__str__()}, кол-во этажей: {h1.__len__()}.')
+print(f'Название: {h2.__str__()}, кол-во этажей: {h2.__len__()}.')
+
+# __len__
+print(len(h1))
+=======
+class House:
+    def __init__(self, name:str, number_of_floors:int):
+        self.name = name
+        self.number_of_floors = number_of_floors
+
+    def __len__(self):
+        return self.number_of_floors
+    def __str__(self):
+        return self.name
+    def go_to (self, new_floors:int):
+        if new_floors > self.number_of_floors:
+            print ("Такого этажа не существует")
+        else:
+            for i in range(1,new_floors+1):
+                print (i)
+
+# h1 = House('ЖК Горский',18)
+# h2 = House('Домик в деревне',2)
+# h1.go_to(5)
+# h2.go_to(10)
+
+h1 = House('ЖК Эльбрус', 10)
+h2 = House('ЖК Акация', 20)
+
+# __str__
+print(f'Название: {h1.__str__()}, кол-во этажей: {h1.__len__()}.')
+print(f'Название: {h2.__str__()}, кол-во этажей: {h2.__len__()}.')
+
+# __len__
+print(len(h1))
+>>>>>>> origin/master
+print(len(h2))
\ No newline at end of file
